### DISCUSSION TITLE
Any plan to have a python wrapper?

### DISCUSSION THREAD

**Discussion by lfz** [DGtal-team/DGtal#1378] [Ideas] [answered]
Hi, thanks for this powerful toolbox, but I only know how to use python (sorry for that), do you have any plan to build a python wrapper for DGtal?

**Comment by dcoeurjo** [comment]
We have discussed about for many years ;). 
There have been some partial bindings  (eg using boost python bindings) but nothing more. 

In fact, we don’t have a lot of python gurus in the dev team.. 

If you want to participate, feel free 😊

**Comment by dcoeurjo** [comment]
Quick update on that issue, @phcerdan is actually working on a python binding 🎉

**Comment by dcoeurjo** [comment, marked as answer]
Quick update on this discussion: it is still very beta but a first python binding of a chunk of DGtal has just been published 🎉 
Documentation and examples will come soon but you can now `pip install dgtal`

kudos @phcerdan

**Reply by copyme** [reply]
Wow! That is great news. Well done :-)

**Reply by davelajoie** [reply]
such a great news! tx
